app/controllers/users_controller.rb

```diff
@@ -0,0 +1,75 @@
+class UsersController < ApplicationController
+  before_action :set_user, only: [:show, :edit, :update, :destroy]
+
+  # GET /users
+  # GET /users.json
+
+  def index
+    @users = User.all
```

Comment:
Not critical now, but you may want to consider adding an active scope here. Probably an HR violation to show inactive users in an open index. https://api.rubyonrails.org/classes/ActiveRecord/Scoping/Named/ClassMethods.html